openapi.yaml

```diff
@@ -0,0 +1,787 @@
+    get:
+      operationId: listGrants
+      tags:
+        - "grants"
+      parameters:
+        - $ref: "#/components/parameters/pagingOffset"
+        - $ref: "#/components/parameters/pagingLimit"
+        - name: "permissionSetId"
+          in: "query"
+          description: Limit to the specified permissionset
+          required: false
+          schema:
+            type: string
+        - name: "securableId"
+          in: "query"
+          description: Limit to the specified securable
+          required: false
+          schema:
+            type: string
+      responses:
+        "200":
+          description: "Display retrieved Grants"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/GrantsResponse"
+    post:
+      operationId: createGrant
+      tags:
+        - "grants"
+      responses:
+        "200":
+          description: "Display created Grant"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/GrantsResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "500":
+          $ref: "#/components/responses/ServerError"
+      requestBody:
+        required: true
+        content:
+          'application/json':
+            schema:
+              $ref: "#/components/schemas/GrantsRequest"
+  /grants/{grantId}:
+    parameters:
+      - name: "grantId"
+        in: "path"
+        description: "Id of Grant to return"
+        required: true
+        schema:
+          type: "string"
+    get:
+      operationId: readGrant
+      tags:
+        - "grants"
+      responses:
+        "200":
+          description: "Display Grant"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/GrantResponse"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        "500":
+          $ref: "#/components/responses/ServerError"
+    put:
+      tags:
+        - "grants"
+      operationId: updateGrant
+      requestBody:
+        required: true
+        content:
+          'application/json':
+            schema:
+              $ref: "#/components/schemas/GrantRequest"
+      responses:
+        "200":
+          description: "Display updated Grant"
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/GrantResponse"
+        "400":
+          $ref: "#/components/responses/BadInput"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        "500":
+          $ref: "#/components/responses/ServerError"
+
+    delete:
+      operationId: deleteGrant
+      tags:
+        - "grants"
+      responses:
+        "200":
+          description: "Grant deleted"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        "500":
+          $ref: "#/components/responses/ServerError"
+  /securables:
```

Comment:
What does the endpoint that looks up children of folders look like in the swagger?  https://dominodatalab.atlassian.net/browse/DOM-38848